lib/tasks/user.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+namespace :user do
+  desc "migrate user registered_at"
```

Comment:
@hengsokly please kindly provide a fully describe what is the reason for this rake task